frontend/model/notifications/templates.js

```diff
@@ -145,17 +145,33 @@ export default ({
     }
   },
   NEW_PROPOSAL (data: { groupID: string, creatorID: string, proposalHash: string, subtype: NewProposalType }) {
-    const args = {
-      name: `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${data.creatorID}`,
-      ...LTags('strong')
-    }
+    const isCreator = data.creatorID === sbp('state/vuex/getters').ourIdentityContractId // notification message is different for creator and non-creator
+    const args = isCreator
+      ? null
+      : {
+          name: `${CHATROOM_MEMBER_MENTION_SPECIAL_CHAR}${data.creatorID}`,
+          ...LTags('strong')
+        }
+
     const bodyTemplateMap = {
-      ADD_MEMBER: L('{strong_}{name}{_strong} proposed to add a member to the group. Vote now!', args),
-      CHANGE_MINCOME: L('{strong_}{name}{_strong} proposed to change the group mincome. Vote now!', args),
-      CHANGE_DISTRIBUTION_DATE: L('{strong_}{name}{_strong} proposed to change the group distribution date. Vote now!', args),
-      CHANGE_VOTING_RULE: L('{strong_}{name}{_strong} proposed to change the group voting system. Vote now!', args),
-      REMOVE_MEMBER: L('{strong_}{name}{_strong} proposed to remove a member from the group. Vote now!', args),
-      GENERIC: L('{strong_}{name}{_strong} created a proposal. Vote now!', args)
+      ADD_MEMBER: isCreator
+        ? L('you proposed to add a member to the group.')
```

Comment:
Every sentence that starts with "you" needs to be replaced with "You" because sentences should start with an uppercase letter.